lib/trustdevice_pro_plugin.dart

```diff
@@ -27,6 +35,383 @@ class TrustdeviceProPlugin {
   }
 }
 
+class TDRiskConfiguration {
+  TDRiskConfiguration({
+    required this.partner,
+    required this.appKey,
+    required this.appName,
+    required this.country,
+    this.debug,
+    this.timeLimit,
+    this.location,
+    this.collectLevel,
+    this.android,
+    this.ios,
+    this.captcha,
+    this.customConfiguration,
```

Comment:
> this commit is not universal
@lurenman thanks for reply! did you mean to add some unhandled options? so I add a `customConfiguration` to handle some unexpected situations.